mobx/lib/mobx.dart

```diff
@@ -64,4 +64,6 @@ export 'package:mobx/src/core.dart'
         DerivationState;
 export 'package:mobx/src/core/atom_extensions.dart';
 
-export 'version.dart';
+
+/// The current version as per `pubspec.yaml`
+const version = '2.1.0';
```

Comment:
this has been removed and moved to a separate `version.dart` file. Please remove this and move to that. You can run `melos run set_version` to sync up with `pubspec.yaml`